### ISSUE TITLE
Powershell for SQL Server Reporting Services

### ISSUE THREAD

**Issue by ddominici** [DevMarche/eventi#7]
Ripropongo anche una sessione che terrò al SQL Saturday di Parma, con qualche modifica...
Automatizzare il deploy dei reports, modificarne il contenuto a runtime, cambiare le connessioni alle fonti dati, inviare in maniera temporizzata i reports nel formato desiderato. Il tutto utilizzando solo un linguaggio: Powershell. Beh, due, dai... Anche un pizzico di T-SQL non guasta...

**Comment by nonno** [comment]
Per attirare l'attenzione delle masse forse sei andato un po' troppo nello specifico, così come l'hai impostato sembra più una cosa da specialisti.
Nel mio piccolo posso dirti che mi sono reso conto della potenza e versatilità di Powershell solo pochi giorni fa, secondo me se abbassi un po' il tiro di interessati ne trovi. Basta spiegare che gli script si possono lanciare anche su Mac e Linux (installando un paio di robette) e che con Visual Studio Code è un piacere editarli. Diciamo che per automatizzare certi processi forse è più appropriato del bash...

Che formato hai in mente? Techbar serale con pizza come al solito o qualcosa di diverso? Preferenze sulla zona?

**Comment by ddominici** [comment]
In effetti questa sessione è molto DBA-oriented, ma era pensata per un evento SQL Server only. La tua proposta non è male, potrebbe essere un Techbar in cui vediamo le basi di Powershell e un po' di automazione su Windows e SQL Server...

**Comment by nonno** [comment]
Diciamo che uno strumento di scripting "potente" fa comodo a tutti; io Powershell non lo conosco per niente ma approfondirei volentieri, e magari ce ne sono altri, specialmente se riesci a metterlo a confronto con bash. Una sessione del genere la potresti riproporre facilmente anche in qualche conferenza "generalista".

**Comment by ste8** [comment]
concordo con la linea più "generalista" su cui state convergendo... potrebbe essere un bell'argomento mai trattato in un evento DevMarche :)

**Comment by nonno** [comment]
@ddominici naturalmente il mio è un tentativo di dargli una forma che a senso potrebbe riscuotere più successo in quanto più utile per le masse, non è che tu debba allontanarti da ciò che avevi in mente. Valuta tu, io e @ste8 volendo abbiamo anche un collega che con Powershell va decisamente forte, tante volte avessi bisogno di un supporto lunedì magari glielo facciamo presente. Io in caso non avrei problemi a occuparmi dell'organizzazione.

**Comment by bizcomit** [comment]
io lo trovo interessante e come dice @ste8, mai trattato in devmarche. 
come location, ospito volentieri :)
